plugin-build/src/test/resources/testFixtures/dummy-bin/dummy-sentry-cli

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "This is just a dummy script"
```

Comment:
should we create and delete on exit (per test or test class) instead of adding this file to the repo?

Reply:
Yup fixed